pipelines/kubeflow_pipelines_component_examples/filter_file_component/drop_even_rows.pipeline

```diff
@@ -54,28 +55,87 @@
               }
             }
           ]
+        },
+        {
+          "id": "67a92256-96f5-480a-8140-fbe0d3c2c30f",
+          "type": "execution_node",
+          "op": "component_Filtertextusingshellandgrep",
```

Comment:
The op identifier `component_Filtertextusingshellandgrep` doesn't seem to match the identifier that the registry produces for the included sample component, which appears to be `filter_text_using_shell_and_grep_Filtertext`. Therefore this pipeline won't work out of the box. We need to rebuild the sample pipeline to address the issue.